marklogic-data-hub/src/test/ml-modules/root/test/suites/data-hub/5/data-services/lib/mappingService.sjs

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+// Utils for getDocumentForTesting's tests.
+const DocumentForTestingUtils = {
+  STEP_NAME: 'prospect2CustomerMappingStep',
+  DATABASE_NAME: 'data-hub-FINAL',
+  // Load data not provided via test-data directory.
+  loadTestData: function () {
+    // Found it difficult to figure out how to load a mapping step via test-data directory and get it into the collection
```

Comment:
@srinathgit  Can you figure out the problem here? I know we have lots of ML tests that load mappings